Add instructions to delete MKE backup (#3212)
mesosphere/dcos-docs-site: pages/mesosphere/dcos/services/kubernetes/2.7.0-1.18.6/operations/disaster-recovery/index.md

<p class="message--important"><strong>IMPORTANT: </strong> This command does not manage S3 buckets, so its usage should be monitored by you.</p>

## View backup log messages

> To diagnose a failed MKE cluster backup, you can view the log files for the Kubernetes pod that was launched to perform the backup. Use these steps:

Reviewer: ```suggestion
To diagnose a failed MKE cluster backup, you can view the log files for the Kubernetes pod launched to perform the backup. Use these steps:
```

1. While the "dcos kubernetes cluster backup" command is running, get the pod id of the pod that is trying to do the ark backup.

```shell
$ kubectl get pods -n heptio-ark
```

2. Check the log file of the running `heptio-ark` pod that is attempting to do the backup. Use this command and replace `<pod-id>` with the pod id given by the previous command.

```shell
$ kubectl logs -f -n heptio-ark <pod-id>
```

> The `-f` option will "follow" the log file thus you will see all the messages, including any error messages that help determine the cause of the backup failure.

Reviewer: ```suggestion
The `-f` option will "follow" the log file thus you will see all the messages, including any error messages that assist determining the cause of the backup failure.
```

## Remove backup entries

> If you no longer need a backup entry, you can remove it of the Kubernetes cluster with the following steps.

Reviewer: ```suggestion
If you no longer need a backup entry, you can remove it from the Kubernetes cluster with the following steps.
```
